build.sbt

```diff
@@ -1,54 +1,11 @@
+import org.scalajs.linker.interface.ModuleSplitStyle
 import sbtcrossproject.crossProject
 import sbtcrossproject.CrossType
 import Settings.Libraries._
 
-val reactJS                   = "16.13.1"
-val FUILess                   = "2.8.7"
-lazy val kindProjectorVersion = "0.11.3"
-
-parallelExecution in (ThisBuild, Test) := false
-
-cancelable in Global := true
-
-addCommandAlias(
-  "exploreWDS",
-  "; stopWDS; explore/fastOptJS::startWebpackDevServer; ~explore/fastOptJS"
-)
-
-addCommandAlias(
-  "constraintsWDS",
-  "; constraints/fastOptJS::stopWebpackDevServer; constraints/fastOptJS::startWebpackDevServer; ~constraints/fastOptJS"
-)
-
-addCommandAlias(
-  "targeteditorWDS",
-  "; stopWDS; targeteditor/fastOptJS::startWebpackDevServer; ~targeteditor/fastOptJS"
-)
-
-addCommandAlias(
-  "observationtreeWDS",
-  "; observationtree/fastOptJS::stopWebpackDevServer; observationtree/fastOptJS::startWebpackDevServer; ~observationtree/fastOptJS"
-)
-
-addCommandAlias(
-  "proposalWDS",
-  "; proposal/fastOptJS::stopWebpackDevServer; proposal/fastOptJS::startWebpackDevServer; ~proposal/fastOptJS"
-)
-
-addCommandAlias(
-  "stopWDS",
-  "fastOptJS::stopWebpackDevServer"
-)
-
-addCommandAlias(
-  "quickTest",
-  "modelJVM/test"
-)
-
-addCommandAlias(
-  "fixImports",
-  "; scalafix OrganizeImports"
-)
```

Comment:
Can we keep `quickTest` and `fixImports`?

Reply:
Right, I deleted too much